cognite/neat/utils/xml_.py

```diff
@@ -35,3 +36,39 @@ def get_children(element: Element, child_tag: str, no_children: int = -1) -> lis
             else:
                 children.append(child)
     return children
+
+
+def get_children_from_tag(
```

Comment:
Added a new method for this because I don't want to deal with the namespace prefix, and leveraging elementTree.findall() for this purpose.

Reply:
Great! remove old `get_children` and renamed `get_children_from_tag` to `get_children`